ufc/bandit-flags-v1.json

```diff
@@ -1,158 +1,207 @@
 {
     "createdAt": "2024-04-17T19:40:53.716Z",
     "environment": {
-      "name": "Test"
+        "name": "Test"
```

Comment:
my editor forced consistent 4space spacing across the document so most of this diff is noise